install/install.go

```diff
@@ -450,7 +503,24 @@ func main() {
 		log.Fatalf("Validating supplied configuration failed. Exiting...")
 	}
 
-	generateCertificate()
+	if data.tlsGenerate {
+		if data.organization == "" {
+			log.Fatalf("Certificate generation requested but -org needs to be provided to generate certificate")
```

Comment:
I think usually we can use default org here, e.g, if user does care about the certificate, they should apply one from CA, instead of generate one.